content/builds/heal-alac-willbender.md

```diff
@@ -0,0 +1,102 @@
++++
+title = "Symbolic Healbender"
+description = "This build is designed around the hammer Auto Attack, being able to proc Relic of the Karakosa consecutively with a burst heal rotation. Besides being easy to play, this build has flexible utilities, decent crowd control and excellent condi cleanses."
+date = 2024-04-02
+draft = false
+template = "build.html"
+
+[taxonomies]
+tags = ["heal","alacrity","guardian","willbender","pof","lowcog","lowphys","lowrep"]
+authors = ["xellink"]
+
+[extra]
+series = "guardian"
+tagline = "Auto-Attack Healbender"
+keywords = "Guild Wars 2, GW2, LI, willbender"
+toc = true
+spec = "dragonhunter"
+balance = "March 2024"
++++
+
+This is a very low intensity build that requires minimal effort to play, where majority of its effects are based on auto-attacks. The end of the auto-attack chain {{ skill(id="9161") }} rely on {{ trait(id="558") }} to provide heals.
+
+This build is low cognition, low physical complexity, has very few repetitions and is functional in instanced content. This build requires only one weapon with no requirement of a weapon swap: Hammer!
+
+
+## Gearing and Consumables
+**Equipment**
+- Full Harrier's 
+- Hammer
+
+**Weapon Swap**
+Weapon swap is optional, all the available options will lock you out of hammer and thus drop your boon coverage. Alternate weapon sets should be prioritised based on utility requirements. 
+
+**Runes**
+- Superior Rune of the Monk
+- Superior Rune of Rebirth (budget)
+
+**Sigils**
+- Superior Sigil of Transference
+- Superior Sigil of Concentration
+- Superior Sigil of Water (Budget Alternative)
+
+**Relic**
+- Relic of Karakosa
+
+**Food**
+  - Bowl of Fruit Salad with Mint Garnish
```

Comment:
Did you want to indent these? I'm not even sure if it'll do anything, but it might make the bullet points inconsistent.